meshapp/src/pages/Profiles/Page.tsx

```diff
@@ -20,22 +13,6 @@ import { axiosInstance } from "src/config/axios-config";
 // New Imports
```

Comment:
i know you didn't add this comment, but this is a pointless comment, could you remove it  lol

Reply:
yeah for sure